src/main/java/io/cdap/plugin/servicenow/connector/ServiceNowConnectorConfig.java

```diff
@@ -172,17 +180,23 @@ public Double convertToDoubleValue(Object fieldValue) {
     if (fieldValue instanceof String && Strings.isNullOrEmpty(String.valueOf(fieldValue))) {
       return null;
     }
-
-    return Double.parseDouble(String.valueOf(fieldValue));
+    try {
+      return NumberFormat.getNumberInstance(java.util.Locale.US).parse(String.valueOf(fieldValue)).doubleValue();
```

Comment:
make sure that `US` local parsing is expected to different regions or better to have a comment stating that US locale number format is hardcoded.